src/applications/edu-benefits/complaint-tool/reducers/schoolSelect.js

```diff
@@ -74,19 +78,20 @@ export default function schoolSearch(state = initialState, action) {
         searchResultsCount,
         showInstitutions,
         showInstitutionsLoading,
+        showNoResultsFound,
         showPagination,
         showPaginationLoading
       };
     }
 
-    case SEARCH_INPUT_CHANGE: {
+    case SEARCH_INPUT_CHANGED: {
       return {
         ...state,
         searchInputValue: action.searchInputValue
       };
     }
 
-    case SELECT_INSTITUTION: {
+    case INSTITUTION_SELECTED: {
```

Comment:
Address [this](https://github.com/department-of-veterans-affairs/vets-website/pull/8062#discussion_r206716897).